src/epyq/_build.py

```diff
@@ -1,4 +1,4 @@
-# This file should be overwritten when a distribution build is done
+# This file has been generated

```

Comment:
Do we need to change this comment? I think the previous comment is more appropriate. With the new comment, it almost seems like the file was accidentally checked in.